code/modules/vore/eating/digest_act_vr.dm

```diff
@@ -49,7 +49,7 @@
 	. = ..()
 
 /obj/item/weapon/card/id/digest_act(var/atom/movable/item_storage = null)
-	desc = "A partially digested card that has seen better days. Much of it's data has been destroyed, but the card itself is still completely salvageable. Please recycle it at crew resources office."
+	desc = "A partially digested card that has seen better days. The damage appears to be only cosmetic, but the access codes need to be reprogrammed at the HoP office."
```

Comment:
Would be much better if we said, `Much of it's data has been destroyed, but the card itself is still completely salvageable. Please recycle it at the Head of Personnel's office."`

Reply:
I feel like the recycling part and the mention of destruction have been factors behind people considering them trash. I doubt anyone's figured out the "recycling" means just restoring the access instead of getting a replacement and leaving the HoP baffled wondering how the card itself can get turned into materials.

Reply:
It's fine for now. @SpadesNeil can just make another PR to change it if he wants.